### PR TITLE
Implementation of the lexical in headless cms

### PR DESCRIPTION
**Pull request by mihajlovco**
### app-headless-cms package
* new field renderers
Folder /fieldRenderes/LexicalText
- LexicalTextInput
- LexicalTextInputs

### Lexical API updates
- Introduced toolbar action plugins
- Added `styles` prop to lexical editor 

### Additional Lexical fixes
- Fixed placeholder for lexical. With this fix, the placeholder will be in place in HCMS, block variables, and PB
- Added overflow `auto` to the lexical editor for scrolling on max-height

#### Possibility user to add an image
Components :
- ImageComponent
- InsertImageAction
- ImagePayload 
- ImageCommand
- ImageNode

#### Implemented new static toolbar 
New components 
- StaticToolbar
- AddStaticToolbar
- Updated RichTextEditor with staticToolbar API

#### Ability users to add custom plugins to ParagraphEditor and HeadingEditor
- In this way, devs can hook and add plugins directly into the paragraph and header components.

## How Has This Been Tested?
Manually.

https://github.com/webiny/webiny-js/assets/82515066/358b48e9-baf0-457f-b063-c12d2b316cb5

### CODE REVIEW

File: packages/lexical-editor-cms-actions/src/components/LexicalCmsEditor.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    theme?: Record<string, any>;
+}
+
+export const LexicalCmsEditor: React.FC<RichTextContentEditorProps> = ({
+    placeholder,
+    toolbarActionPlugins,
+    tag,
+    ...rest
+}) => {
+    const { theme } = usePageElements();
```
Comment (adrians5j) [inline packages/lexical-editor-cms-actions/src/components/LexicalCmsEditor.tsx]: I see `usePageElements` here. Not sure whether what we discussed today with Bruno/Monika was something that we'll do later, or we want to do it now?

If you ask me, it's now.

**Review by adrians5j — CHANGES_REQUESTED** [review]
Left a quick question.

File: packages/lexical-editor-cms-actions/src/components/LexicalCmsEditor.tsx (hunk 1 above)
Comment (mihajlovco) [reply packages/lexical-editor-cms-actions/src/components/LexicalCmsEditor.tsx]: @adrians5j I will implement now

**Comment by mihajlovco** [comment]
Test1 -  /cypress

**Pull request closed without merging**